Vungle/VungleRouter.m

```diff
@@ -774,11 +775,15 @@ - (void)vungleAdPlayabilityUpdate:(BOOL)isAdPlayable
                                                                               adMarkup:adMarkup];
     if (targetDelegate) {
         if (isAdPlayable) {
-            MPLogInfo(@"Vungle: Ad playability update returned ad is playable for Placement ID: %@", placementID);
             [targetDelegate vungleAdDidLoad];
         } else {
             MPLogInfo(@"%@", message);
-            if (!self.isAdPlaying) {
+            // Ignore any playability update if the delegate is the playing ad
```

Comment:
I'm not sure about Vungle's standard for leaving a comment block. I don't see any comments in the adapter codebase. This is to inform us when developing, but should it be put into the code?